sdk/opendp/whitenoise/evaluation/blackbox/_base.py

```diff
@@ -0,0 +1,32 @@
+from opendp.whitenoise.evaluation.params._privacy_params import PrivacyParams
+from opendp.whitenoise.evaluation.params._eval_params import EvaluatorParams
+from opendp.whitenoise.evaluation.report._report import Report
+from abc import ABC, abstractmethod
+
+class BlackBoxPrivacyInterface(ABC):

```

Comment:
why is this internal?

Reply:
nvm, all of it is so we can test it

Reply:
Ya, I really liked the internal convention which allows me to check-in and then we can make it external when I implement the interfaces and add unit tests that pass. So, for skeleton, keeping it internal for now.